tiledb/cloud/utils.py

```diff
@@ -0,0 +1,24 @@
+import inspect
+import logging
+
+logger = logging.getLogger("tiledb.cloud")
+
+
+def getsourcelines(func):
+    if func is not None and not callable(func):
```

Comment:
oh, btw this can be written as `if not callable(func)` (`callable(None)` -> `False`)